website/common/locales/en/npc.json

```diff
@@ -16,6 +16,7 @@
   "mattBoch": "Matt Boch",
   "mattBochText1": "Welcome to the Stable! I’m Matt, the beastmaster. Every time you complete a task, you'll have a random chance at receiving an Egg or a Hatching Potion to hatch Pets. When you hatch a Pet, it will appear here! Click a Pet's image to add it to your Avatar. Feed them with the Pet Food you find, and they'll grow into hardy Mounts.",
   "welcomeToTavern": "Welcome to The Tavern!",
+  "sleepHeader": "Pause Damage",
```

Comment:
This text already exists as `pauseDailies`. Unless there's reason to believe it should be translated in two different ways, we should try to reuse the token when it contains the same text.